runtime/mgmt/mgmtinit.c

```diff
@@ -31,3 +33,48 @@ JNI_OnLoad(JavaVM *vm, void *reserved)
 {
 	return JNI_VERSION_1_8;
 }
+
+/**
+ * Determines whether file path has user permissions only.
+ *
+ * @param path file path
+ * @return Returns false on Windows and true if the file has only user permissions,
+ * false otherwise on other operation systems.
+ *
+ * @throws IOException in case of an error
+ */
+JNIEXPORT jboolean JNICALL 
+Java_sun_management_FileSystemImpl_isAccessUserOnly0(JNIEnv *env, jclass c, jstring path)
+{
+	J9VMThread *currentThread = (J9VMThread*)env;
+	J9JavaVM *vm = currentThread->javaVM;
+	J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
+	const char* pathUTF;
+	struct J9FileStat buf;
```

Comment:
This can be declared in the `} else {` block.  Always declare variables in the smallest enclosing scope as it makes it easier for readers to reason about the variables lifecycle